parachain/runtime/kusama-common/Cargo.toml

```diff
@@ -2,8 +2,9 @@
 name = "snowbridge-kusama-common"
 description = "Snowbridge Kusama Common"
 version = "0.0.1"
```

Comment:
I guess we can also bump the version to "0.9.0" now?

Reply:
Done in https://github.com/Snowfork/snowbridge/pull/1101/commits/06b88e3b07d94e63b55f9ccf33a1d4e7b9adbbca.